1652857975934	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1652857976092	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile 304
1652858589673	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1652858590095	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1652858590095	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1652858590098	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1652858590545	Sync.SyncScheduler	DEBUG	Woke from sleep.
1652858590545	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1652858590545	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1652858590546	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1652858592548	Sync.Service	DEBUG	User-Agent: Firefox/100.0.1 (Intel Mac OS X 10.15) FxSync/1.102.0.20220513165813.desktop
1652858592548	Sync.Service	INFO	Starting sync at 2022-05-18 09:23:12 in browser session Kaj7xo9lnzpb
1652858592548	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1652858592555	Sync.Status	INFO	Resetting Status.
1652858592556	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1652858592557	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1652858592557	FirefoxAccounts	DEBUG	getOAuthToken enter
1652858592560	FirefoxAccounts	TRACE	getCachedToken returning cached token
1652858592560	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1652858592561	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1652858592591	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1652858592592	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]"}
1652858592595	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/195039599/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1652858592595	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1652858592596	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1652858592596	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1652858592596	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1652858592599	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
